### PR TITLE
added working instance for Pointer.swift

### PR DESCRIPTION
**Pull request by jasperurbiztondo**
+added working instance for Pointer.swift

### CODE REVIEW

File: cs311l-9365-swift/classes/model/Pointer.swift [hunk 1]
```diff
@@ -1,3 +1,9 @@
 struct Pointer{
-    //To be added by Ramon
+    var x: Int
+    var y: Int
```
Comment (saintramon) [inline cs311l-9365-swift/classes/model/Pointer.swift:3]: Make this private.

File: cs311l-9365-swift/classes/model/Pointer.swift [hunk 2]
```diff
@@ -1,3 +1,9 @@
...
+
+    init(x: Int, y: Int){
+        self.x = x
+        self.y = y
+    }
```
Comment (saintramon) [inline cs311l-9365-swift/classes/model/Pointer.swift:8]:
```suggestion
    }
    
    public func getX() -> Int {
        return x
    }

    public func getY() -> Int {
        return y
    }
    
```

**Review by saintramon — DISMISSED** [review]
Looks good to me, just a few minor changes.

File: cs311l-9365-swift/classes/model/Pointer.swift (hunk 1 above)
Comment (saintramon) [inline cs311l-9365-swift/classes/model/Pointer.swift]: Make this private.

File: cs311l-9365-swift/classes/model/Pointer.swift (hunk 2 above)
Comment (saintramon) [inline cs311l-9365-swift/classes/model/Pointer.swift:8]:
```suggestion
    }
    
    public func getX() -> Int {
        return x
    }

    public func getY() -> Int {
        return y
    }
```

**Review by saintramon — CHANGES_REQUESTED** [review]
Looks good to me, just a few minor changes.

File: cs311l-9365-swift/classes/model/Pointer.swift (hunk 1 above)
Comment (jasperurbiztondo) [reply cs311l-9365-swift/classes/model/Pointer.swift]: Done. Thank you.

**Pull request merged**